bodhi/tests/server/test_util.py

```diff
@@ -1424,3 +1424,37 @@ def test___call___success(self, Session, log_exception):
         Session.return_value.commit.assert_called_once_with()
         Session.return_value.close.assert_called_once_with()
         Session.remove.assert_called_once_with()
+
+
+class TestPyfileToModule(unittest.TestCase):
+    """Test the pyfile_to_module() function."""
+
+    def setUp(self):
+        self.tempdir = tempfile.mkdtemp('bodhi')
+
+    def tearDown(self):
+        shutil.rmtree(self.tempdir)
```

Comment:
I totally agree but since I mostly moved the test cases from `consumers` to `tasks`, I wanted to keep those changes to a minimum so git can follow the rename. I can totally fix that in a followup commit though.

Reply:
Sure. :+1:

Reply:
See #3357.